Bomb.cpp

```diff
@@ -435,9 +435,8 @@ void Bomb::MarkExplosionXSideInMap(int lenght, bool right, char ch)
 {
 	int sign = right ? 1 : -1;
 	MatPos pos = matPos;
-	pos.l += 1 * sign;
 
-	while (lenght)
+	while (lenght >= 0)
```

Comment:
nice catch ;) 👏

Reply:
haha :)) i m a genius :))

Reply:
:))) i think you're also the guy who wrote these methods

Reply:
xD